bindings/pyroot/pythonizations/test/CMakeLists.txt

```
@@ -125,8 +125,19 @@ if (dataframe)
     ROOT_ADD_PYUNITTEST(pyroot_rdfdescription rdfdescription.py)
 endif()
 
-# RTensor pythonizations
+# SOFIE-GNN pythonizations
 if (dataframe AND tmva)
+    if(NOT MSVC OR CMAKE_SIZEOF_VOID_P EQUAL 4 OR win_broken_tests AND PYTHON_VERSION_MAJOR_Development_Main EQUAL 3 )
+        find_python_module(sonnet QUIET)
+        find_python_module(graph_nets QUIET)
+        if (PY_SONNET_FOUND AND PY_GRAPH_NETS_FOUND)
+            ROOT_ADD_PYUNITTEST(pyroot_pyz_sofie_gnn sofie_gnn.py PYTHON_DEPS numpy sonnet graph_nets)
+        endif()
+    endif()
+endif()
+
+# RTensor pythonizations
+if (tmva AND sofie)
```

```suggestion
if (tmva AND tmva-sofie)
```

`sofie` doesn't exist in CMake